packages/app-desktop/gui/Sidebar/SidebarHeaderBar.tsx

```
@@ -0,0 +1,53 @@
+};
+
+const onToggleAllFolders = (allFoldersCollapsed: boolean) => {
+	void CommandService.instance().execute(
+		'toggleAllFolders',
+		!allFoldersCollapsed,
+	);
+};
+
+interface Props {
+	allFoldersCollapsed: boolean;
+	containerRef: (el: HTMLDivElement | null)=> void;
```

ESLint

[error] 'el' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)